### PR TITLE
Status Update 2020-07-10

### PR DESCRIPTION
**Pull request by adrianiurca**

### CODE REVIEW

File: _posts/status_update_2020_07_10.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+The IAC team has been working on a lot of things over the past week.
+We want to share with our community some of the most interesting things.
+Here’s a brief overview of what we’ve been up to:
+
+## Module Releases from IAC Team
+
+- [puppetlabs/iis](https://github.com/puppetlabs/puppetlabs-iis)(7.1.0)
+- [puppetlabs/java_ks](https://github.com/puppetlabs/puppetlabs-tomcat)(4.1.0)
```
Comment (sheenaajay) [inline _posts/status_update_2020_07_10.md]:
```suggestion
- [puppetlabs/tomcat](https://github.com/puppetlabs/puppetlabs-tomcat)(4.1.0)
```

File: _posts/status_update_2020_07_10.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Customer tickets this week
+
+The team works hard at Docker issue ([MODULES-10691](https://tickets.puppetlabs.com/browse/MODULES-10691)).
+We have a PR opened for this issue [PR](https://github.com/puppetlabs/puppetlabs-docker/pull/632).
+Many thanks to [Daiana](https://github.com/daianamezdrea) for managing this problem.
+
+## Michael and Sheena are getting back to DCS revamp work after a hard triage
+
+@TODO
```
Comment (sheenaajay) [inline _posts/status_update_2020_07_10.md:44]: @michaeltlombardi Could you please confirm this . 
Will be working on a handcrafted automated end-to-end test which validates autogenerated modules are ready for production.

File: _posts/status_update_2020_07_10.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+layout: post
+title: "10th July 2020: IAC Team Status Update"
+author: adrianiurca
+categories:
+
+- team
+- status
+  tags:
```
Comment (sanfrancrisko) [inline _posts/status_update_2020_07_10.md:8]:
```suggestion
tags:
```

File: _posts/status_update_2020_07_10.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+- [puppetlabs/tomcat](https://github.com/puppetlabs/puppetlabs-tomcat)(4.1.0)
+- [puppetlabs/powershell](https://github.com/puppetlabs/puppetlabs-powershell)(4.0.0)
+
+## Community Shout outs for IAC
+
+We appreciate our community contributors as they are always ready to help and they have excellent ideas. Here are some highlights of this week:
+
+Thanks to [Ben Ford](https://github.com/binford2k) for his great idea [feature](https://github.com/puppetlabs/community_management/pull/39) on community_management.
+
+Thanks to [Chris Denneen](https://github.com/cdenneen) for the contributions on pdk-templates.
```
Comment (sanfrancrisko) [inline _posts/status_update_2020_07_10.md:34]: I would perhaps link to the PRs from Chris too

File: _posts/status_update_2020_07_10.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+Many thanks to [Daiana](https://github.com/daianamezdrea) for managing this problem.
+
+## Michael and Sheena are getting back to DCS revamp work after a hard triage
+
+@TODO
+
+## Happy 90 days at Puppet
+
+A great achievement made by [Adrian](https://github.com/adrianiurca).
+For his hard work made in these 3 months, he was rewarded to write this blog post.
```
Comment (carabasdaniel) [inline _posts/status_update_2020_07_10.md:49]:
```suggestion
For the great progress he made in this time, as a reward, he got to write this week's status update.
```

File: _posts/status_update_2020_07_10.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+- releases
+- community_management
+- iis
+- tomcat
+- powershell
+- shoutouts
+
+---
+
+The IAC team has been working on a lot of things over the past week.
```
Comment (carabasdaniel) [inline _posts/status_update_2020_07_10.md:18]:
```suggestion
This has been as short week for the IAC team and we mostly focused our attention on maintenance and support activities.
```

File: _posts/status_update_2020_07_10.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+Thanks to [Chris Denneen](https://github.com/cdenneen) for the contributions on pdk-templates([#338](https://github.com/puppetlabs/pdk-templates/pull/338), [#337](https://github.com/puppetlabs/pdk-templates/pull/337), [#332](https://github.com/puppetlabs/pdk-templates/pull/332))
+
+## Customer tickets this week
+
+The team works hard at Docker issue ([MODULES-10691](https://tickets.puppetlabs.com/browse/MODULES-10691)).
+We have a PR opened for this issue [PR](https://github.com/puppetlabs/puppetlabs-docker/pull/632).
+Many thanks to [Daiana](https://github.com/daianamezdrea) for managing this problem.
+
+## Michael and Sheena are getting back to DCS revamp work after a hard triage
```
Comment (michaeltlombardi) [inline _posts/status_update_2020_07_10.md:41]:
```suggestion
## Michael and Sheena are getting back to DSC revamp work after a hard triage
```

File: _posts/status_update_2020_07_10.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+We want to share with our community some of the most interesting things.
+Here’s a brief overview of what we’ve been up to:
+
+## Module Releases from IAC Team
+
+- [puppetlabs/iis](https://github.com/puppetlabs/puppetlabs-iis)(7.1.0)
+- [puppetlabs/tomcat](https://github.com/puppetlabs/puppetlabs-tomcat)(4.1.0)
+- [puppetlabs/powershell](https://github.com/puppetlabs/puppetlabs-powershell)(4.0.0)
+
+## Community Shout outs for IAC
```
Comment (michaeltlombardi) [inline _posts/status_update_2020_07_10.md:27]:
```suggestion
## Community Shoutouts for IAC
```

File: _posts/status_update_2020_07_10.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+We appreciate our community contributors as they are always ready to help and they have excellent ideas. Here are some highlights of this week:
+
+Thanks to [Ben Ford](https://github.com/binford2k) for his great idea [feature](https://github.com/puppetlabs/community_management/pull/39) on community_management.
+
+Thanks to [Chris Denneen](https://github.com/cdenneen) for the contributions on pdk-templates([#338](https://github.com/puppetlabs/pdk-templates/pull/338), [#337](https://github.com/puppetlabs/pdk-templates/pull/337), [#332](https://github.com/puppetlabs/pdk-templates/pull/332))
+
+## Customer tickets this week
```
Comment (michaeltlombardi) [inline _posts/status_update_2020_07_10.md:35]:
```suggestion
## Customer Tickets this Week
```

**Pull request merged**